test/extended/prometheus/prometheus.go

```diff
@@ -580,7 +580,11 @@ var _ = g.Describe("[sig-instrumentation] Prometheus", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 
 			g.By("verifying a service account token is able to authenticate")
-			err = expectBearerTokenURLStatusCodeExec(ns, execPod.Name, fmt.Sprintf("%s/graph", url), bearerToken, 200)
+			err = expectBearerTokenURLStatusCodeExec(ns, execPod.Name, fmt.Sprintf("%s/api/v1/targets", url), bearerToken, 200)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("verifying a service account token can't access prometheus UI")
+			err = expectBearerTokenURLStatusCodeExec(ns, execPod.Name, fmt.Sprintf("%s/graph", url), bearerToken, 404)
```

Comment:
we have a chicken and egg problem since the test won't pass until https://github.com/openshift/cluster-monitoring-operator/pull/1532 merges

Reply:
```suggestion
```